### DISCUSSION TITLE
Prometheus metrics integration

### DISCUSSION THREAD

**Discussion by ManojBhamsagar-Draup** [FlareSolverr/FlareSolverr#788] [Ideas]
We can also think about adding prometheus metrics. I'm using flaresolverr in production and hitting around 1 million urls per day. But I have modified this project to expose prometheus metrics on a http server. Can we implement this in flaresolverr so that it will be useful for people who are using at a bigger scale.

**Comment by ngosang** [comment]
I have an issue for that => https://github.com/FlareSolverr/FlareSolverr/issues/107
PRs are welcome.

**Reply by ManojBhamsagar-Draup** [reply]
I will try to raise pr in sometime

**Reply by ManojBhamsagar-Draup** [reply]
Hey @ngosang, can you please tell me the reasons for migrating from puppeteer to selenium and also the browser from firefox to chrome.

**Reply by ngosang** [reply]
Puppeter and Firefox are detected by Cloudflare :smile:

**Reply by ManojBhamsagar-Draup** [reply]
oh okay, thanks for replying

**Reply by ManojBhamsagar-Draup** [reply]
Is headless puppeteer with chrome good option, because rendering time will be less here. Were you getting banned response using that? And how about using headless selenium.

**Reply by ngosang** [reply]
It's detected. You have the FlareSolverr v1 code here with Puppeter + Chrome => https://github.com/FlareSolverr/FlareSolverr1

**Reply by ManojBhamsagar-Draup** [reply]
Thanks for the info @ngosang

**Reply by ManojBhamsagar-Draup** [reply]
Hi @ngosang, can we use multiple tabs in selenium like concurrecny_context feature in puppeteer?

**Reply by ngosang** [reply]
I don't think so. Each request will have his own browser.

**Reply by ManojBhamsagar-Draup** [reply]
okay
